syntax = "proto3";

package sensors;

message SensorData {
  string sensor_id = 1;
  double temperature = 2;
  double pressure = 3;
  double humidity = 4;
  string timestamp = 5;
}

message Empty {}

service SensorService {
  rpc SendTelemetryData (SensorData) returns (Empty) {}
}